packages/cli/src/utils/validate-fs.ts

```diff
@@ -33,11 +32,10 @@ export function validateOutfileName(filename: string): boolean {
 }
 
 /**
- * Validates a file path.
- * Throws on validation failure
+ * Validates a file path. Throws on validation failure.
  *
- * @param filePath - The file path to validate
- * @returns True if validation succeeded
+ * @param filePath - The file path to validate.
+ * @returns `true` if validation succeeded, `false` otherwise.
```

Comment:
This never returns `false` as it throws instead.

Reply:
3f8379b